apps/dynatrace/base/kustomize.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: dynatrace
+  namespace: flux-system
+spec:
+  path: ./apps/dynatrace/${ENVIRONMENT}/base
+  postBuild:
+    substitute:
+      NAMESPACE: "dynatrace"
+      TEAM_NOTIFICATION_CHANNEL: "dtspo-yellow"
```

Comment:
I dont think it should be `dtspo-yellow` - @hannah38  ?

Reply:
I was going to keep as this during my testing then I can revert back to whatever is recommend.

Reply:
platops-build-notices - configured to be this.